Example/Tests/ILGClasses_Tests_ObjC.m

```diff
@@ -35,8 +41,17 @@ - (void)testNonIncludedTestWorks
                                                    ]];
     
     NSSet *retrievedClasses = [ILGClasses classesPassingTest:^BOOL(Class class) {
-        NSString *className = NSStringFromClass(class);
-        return [className containsString:@"Class1"];
+        if ([ILGClasses class:class orAnyOfItsSuperclassesPassesTest:^BOOL(Class  _Nonnull class) {
+            return class == [NSObject class];
+        }]) {
```

Comment:
![post-64231-this-is-fine-dog-fire-comic-im-n7mp](https://cloud.githubusercontent.com/assets/1976498/13412467/0e4ae4de-df09-11e5-9187-23c7776308c6.png)